README.md

```diff
@@ -173,40 +173,96 @@ Create the PR, using the upstream (parent) branch from `.git/machete` as the bas
 git machete github create-pr [--draft]
 ```
 
-Synchronize with the remote repository: checkout your open PRs, delete unmanaged branches and also delete untracked managed branches with no downstream branch: <br/>
-```shell script
-git machete github sync
-```
-
 **Note**: for private repositories, a GitHub API token with `repo` access is required.
 This will be resolved from the first of:
 1. The `GITHUB_TOKEN` env var.
-2. The content of the .github-token file in the home directory (`~`). This file has to be manually created by the user.
+2. The content of the `.github-token` file in the home directory (`~`). This file has to be manually created by the user.
 3. The auth token from the current [`gh`](https://cli.github.com/) configuration.
 4. The auth token from the current [`hub`](https://github.com/github/hub) configuration.
 
 <br/>
 
+
+## FAQ
+
+#### I've run `git machete discover`... but the branch layout I see in `.git/machete` doesn't exactly match what I expected. Am I doing something wrong?
+
+[//]: # (For how to find Medium header anchors, see https://www.freecodecamp.org/news/how-to-link-to-a-specific-paragraph-in-your-medium-article-2018-table-of-contents-method-e66595fea549/)
+No! It's all right, `discover` is based on an (imperfect)
+[heuristic](https://medium.com/virtuslab/git-machete-strikes-again-traverse-the-git-rebase-jungle-even-faster-with-v2-0-f43ebaf8abb0#0544)
+which usually yields branch layout close to what the user expected...
+but it still might differ in details (like, branches discovered to be children of `main`/`develop` instead of each other).
```

Comment:
Overall I like the casual style of FAQ. However, I'd slightly reword this to avoid unnecessary parenthesis. Maybe even split into shorter sentences.

Maybe sth like this:
```suggestion
No! It's all right, `discover` is based on an (imperfect)
[heuristic](https://medium.com/virtuslab/git-machete-strikes-again-traverse-the-git-rebase-jungle-even-faster-with-v2-0-f43ebaf8abb0#0544)
which usually yields branch layout close to what the user would expect.
It still might not be perfect and — for example — declare branches to be children of `main`/`develop` instead of themselves.
```

Reply:
Hmm I'd stick to `each other` rather than `themselves`, as `themselves` suggests that _branch X is a child of X itself_... WDYT?

Reply:
yes